### ISSUE TITLE
compiling issues related to overlays

### ISSUE THREAD

**Issue by alec801** [maxnet/berryboot#288]
I am trying to compile berryboot but I keep running into an error when it runs.

I am cloning the repo with 

> sudo git clone --depth=1 https://github.com/maxnet/berryboot.git

I am building with the command 

> sudo ./build-berryboot.sh device_pi1

After several hours I get the error

> /usr/bin/install: cannot stat ‘/home/alec/berryboot/buildroot-2015.02/output/build/rpi-firmware-master/boot/overlays/_.dtb’: No such file or directory
> package/pkg-generic.mk:223: recipe for target '/home/alec/berryboot/buildroot-2015.02/output/build/rpi-firmware-master/.stamp_images_installed' failed
> make: *_\* [/home/alec/berryboot/buildroot-2015.02/output/build/rpi-firmware-master/.stamp_images_installed] Error 1

so that first line is looking for and .dtb files found in the overlays folder, but in that folder every file has the extension .dtbo not .dtb

Any ideas on how to fix this?

Also, unrelated but to compile for 2 devices, can I run the command 

> sudo ./build-berryboot.sh device1 device2

Thanks in advance :)

**Comment by eriknomitch** [comment]
👍 I am also running into this.

**Comment by ErichDerWikinger** [comment]
same for me, building took over an hour!! :-( 
last release of berryboot is not booting up, remains on rainbow splash forever
Raspberry Pi 2

**Comment by Limeth** [comment]
Having the same issue. I need to build it myself because of #259.

**Comment by ErichDerWikinger** [comment]
ok, my fault - thought I had a rPi2, **but** it was a Raspberry Pi 1 Model B+ 
/facepalm/

Just download the right repo for your Pi and it will work.

**Comment by Limeth** [comment]
@frozeniceman What other repo? Do you mean the `berryboot2.0` branch or the `master` branch? Glad to know you found a solution.

**Comment by ErichDerWikinger** [comment]
@Limeth No, I mean I used `sudo ./build-berryboot.sh device_pi2` before and ran into issues

**Comment by Limeth** [comment]
Oh, maybe I have to run it with `sudo`, because I get the error when running `./rebuild-berryboot.sh device_pi1`.

```
for ovldtb in  /home/limeth/workspace/other/berryboot2/buildroot-2015.02/output/build/rpi-firmware-master/boot/overlays/*.dtb; do /usr/bin/install -D -m 0644 ${ovldtb} /home/limeth/workspace/other/berryboot2/buildroot-2015.02/output/images/rpi-firmware/overlays/${ovldtb##*/} || exit 1; done
/usr/bin/install: cannot stat ‘/home/limeth/workspace/other/berryboot2/buildroot-2015.02/output/build/rpi-firmware-master/boot/overlays/*.dtb’: No such file or directory
make: *** [/home/limeth/workspace/other/berryboot2/buildroot-2015.02/output/build/rpi-firmware-master/.stamp_images_installed] Error 1
```

**Comment by Limeth** [comment]
Using `sudo` didn't help.

**Comment by Limeth** [comment]
Alright, so I tried a bunch of things, like compiling device_pi2 first, compiling on a different machine, installing some more build tools. Nothing seemed to work. Unfortunately, I will have to resort to installing Raspbian manually instead, until there is a clear way of fixing this.

**Comment by Elana4** [comment]
I've managed to get it to compile after dealing with the exact same problem, you need to go into the configs and turn the INSTALL_DTBS=y to n, after you do this, save, run the rebuild command with whichever device you want to build, it will run through the building process, afterwards I turned the DTBS to y again and ran the build command again.
